packages/frontend-api/src/Resources/config/graphql-types/ModelType/Cart/CartInput/RemoveCartInputDecorator.types.yaml

```diff
@@ -0,0 +1,10 @@
+RemoveCartInputDecorator:
+    type: input-object
+    decorator: true
+    inherits:
+        - CartInputDecorator
+    config:
+        fields:
+            cartUuid:
+                type: "Uuid"
+                description: "Cart UUID to be removed"
```

Comment:
After clarifying the behavior,  I would like to extend this description - The API users should be aware of the correct usage - logged in users must not provide the uuid? Or they must pass `null` here? I do not like that, lets talk about it, please